gobblin-service/src/test/java/org/apache/gobblin/service/GobblinServiceManagerTest.java

```diff
@@ -501,28 +578,29 @@ public void testDelete() throws Exception {
 
   @Test (dependsOnMethods = "testDelete")
   public void testGitCreate() throws Exception {
+    URI uri = FlowSpec.Utils.createFlowSpecUri(new FlowId().setFlowGroup(TEST_GROUP_NAME).setFlowName("testGitFlow"));
```

Comment:
is the rename in this PR to ensure it's unique so no collisions w/ other tests?  if so, shall we put the name as a constant up where `TEST_GROUP_NAME` and others are defined?

Reply:
Yes the renames is here to avoid collision. I couldn't figure out how to pass a variable in the actual json (which is where the name of the flow comes from not the file name), so I didn't want to separate the name only to hard code it later.